packages/odie-client/src/components/message/messages-container.tsx

```diff
@@ -9,15 +10,23 @@ import type { CurrentUser } from '../../types/';
 interface ChatMessagesProps {
 	currentUser: CurrentUser;
 }
+const DislikeThumb = () => {
+	return (
+		<div className="chatbox-message__dislike-thumb">
+			<ThumbsDown />
+		</div>
```

Comment:
They styles for this override the message container styles when we show it as the user response. Mainly just the border radius I think?

<img width="104" alt="Screen Shot 2024-10-29 at 11 23 52" src="https://github.com/user-attachments/assets/857d6458-ca05-4be4-8b6e-376279c362dd">

Reply:
I missed the border radius for sure, added.